topics/introduction/tutorials/galaxy-intro-strands/tutorial.md

```diff
@@ -231,39 +229,56 @@ This expands the dataset and shows you information about it, and a preview of it
 
 ![Preview of the data from UCSC Table Browser](../../images/genes_human_chr22_preview.png)
 
-1. The preview tells us  several things:
+From this preview we can obtain a huge amount of useful information:
 1. The dataset has over 4000 regions, meaning that there are over 4000 genes on chromosome 22.
-1. The dataset is in **bed** format.  BED is one of several standard formats for representing genome annotation.  BED is a tabular format that we'll expand on below.  We got BED format because BED was preselected as the output format in the UCSC table browser.
+1. The dataset is in **BED** format, one of several standard for representing genome annotation.  We got BED format because BED was preselected as the output format in the UCSC table browser.
```

Comment:
several standard → several standards

Reply:
Fyi @tnabtaf you can use [suggested changes](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-a-suggested-change) to suggest the changes you want to the author, it can make their life easier :)